src/query/v2/interpreter.cpp

```diff
@@ -1179,14 +1183,15 @@ PreparedQuery PrepareIndexQuery(ParsedQuery parsed_query, bool in_explicit_trans
 
 PreparedQuery PrepareAuthQuery(ParsedQuery parsed_query, bool in_explicit_transaction,
                                std::map<std::string, TypedValue> *summary, InterpreterContext *interpreter_context,
-                               DbAccessor *dba, utils::MemoryResource *execution_memory) {
+                               DbAccessor *dba, utils::MemoryResource *execution_memory,
```

Comment:
Please remove the `DbAccessor` parameter from the function where it is not used anymore.

Reply:
Yes and no, it's used on the `PullPlan`. I would rather create a separate PR and clean up the `db_accessor` alltogether. We won't use this anymore on QE side.